apps/cli/src/diagnostic/DebugLlmReq.ts

```diff
@@ -0,0 +1,29 @@
+import { Timer } from "./Timer";
+
+import { ChatMessageLog, ChatMessage } from "@evo-ninja/agent-utils";
+
+export class DebugLlmReq {
+  constructor(
+    public time: Timer,
+    public chat: ChatMessageLog,
+    public response?: ChatMessage
+  ) { }
+
+  get tokens() {
+    return this.chat["persistent"].tokens +
```

Comment:
I don't think this is necessarily something we need to refactor in this PR, but seems like `DebugLlmReq` shouldn't really know about "persistent" and "temporary" messages (Implementation details of the chat log).

Reply:
Yes agreed, we need to better define what the `ChatMessageLog` used by `Chat` looks like.